policyengine_us/parameters/gov/states/nd/tax/income/rates/separate.yaml

```diff
@@ -4,41 +4,49 @@ brackets:
       2021-01-01: 0
     rate:
       2021-01-01: 0.011
+      2023-01-01: 0
   - threshold:
       2021-01-01: 33_850
       2022-01-01: 34_850
+      2023-01-01: 37_375
     rate:
       2021-01-01: 0.0204
+      2023-01-01: 0.0195
   - threshold:
       2021-01-01: 81_775
       2022-01-01: 84_225
+      2023-01-01: 137_550
     rate:
       2021-01-01: 0.0227
+      2023-01-01: 0.0250
   - threshold:
       2021-01-01: 124_575
       2022-01-01: 128_325
+      2023-01-01: .inf
     rate:
       2021-01-01: 0.0264
+      2023-01-01: 0.0250
   - threshold:
       2021-01-01: 222_500
       2022-01-01: 229_175
+      2023-01-01: .inf
     rate:
       2021-01-01: 0.029
+      2023-01-01: 0.0250
         
 metadata:
   threshold_unit: USD
+  uprating: gov.irs.uprating
```

Comment:
we need to add the uprating only to the thresholds not the rate - see CA as an example 

also please add a screenshot to this thread showing that the thresholds are uprated with inflation

Reply:
same for all parameter files with rates and thresholds which are uprated